README.md

```diff
@@ -9,6 +9,82 @@ This package is available on PyPI using:
 ```bash
 pip install qiskit-sphinx-theme
 ```
+## Configure Left Sidebar
+
+To keep UX/UI similar across different Qiskit packages we strongly encourage the following structure for you sidebar, which can be set in the toctree of your `index.rst`:
+
+```
+.. toctree::
+  :maxdepth: 1
+  :hidden:
+
+   Overview <overview>
+   Getting Started <getting_started>
+   Tutorials <tutorials/index.rst>
+   How-to Guides <how_to/index.rst>
+   API Reference <apidocs/index.rst>
+   Explanations <explanations/index.rst>
+   Release Notes <release_notes>
+   GitHub <https://github.com/Qiskit/your-repo>
+```
+
+The above toctree will render a sidebar that looks like the image below:
+
+<img width="348" alt="Screenshot 2022-11-30 at 4 36 26 PM" src="https://user-images.githubusercontent.com/23662430/204913247-761b8202-1bb2-4233-9451-f180b00b5a12.png">
```

Comment:
The expendable sidebar feature is not shown in the screenshots, probably because the `index.rst` files are empty in the example. We should probably clean up the sample doc and make it more like a template and put in things placeholder content under each pages. For this PR though, can you do something like that locally and update these screenshots to show the expandable feature?

Reply:
That screenshot is not meant to show any expandable items. By default there are no expandable headings unless they explicitly configure them in their `setup.py`. This screenshot is meant to show what is displayed when there is just a toctree defined and the screenshots later down show the same sidebar with the additional expandable headings once they're configured :)